docs/reference/mpremote.rst

```diff
@@ -147,24 +262,44 @@ The full list of supported commands are:
   variable ``$EDITOR``). If the editor exits successfully, the updated file will
   be copied back to the device.
 
-- install packages from :term:`micropython-lib` (or GitHub) using the ``mip`` tool:
+.. _mpremote_command_mip:
+
+- **mip** -- install packages from :term:`micropython-lib` (or GitHub) using the ``mip`` tool:
 
   .. code-block:: bash
 
       $ mpremote mip install <packages...>
 
   See :ref:`packages` for more information.
 
-- mount the local directory on the remote device:
+.. _mpremote_command_mount:
+
+- **mount** -- mount the local directory on the remote device:
 
   .. code-block:: bash
 
       $ mpremote mount [options] <local-dir>
```

Comment:
This is a good improvement - for a very difficult topic to explain! 

I wonder if this should have a short animated gif? To show a simple sequence like:

- ls device - empty
- ls PC - foo.py
- mount, ls device - foo.py

Or maybe we just need to record a video, as we've discussed.

Reply:
Yep. +1 to video.